lib/galaxy/dependencies/pipfiles/default/pinned-requirements.txt

```diff
@@ -165,18 +165,18 @@ schema-salad==2.7.20181126142424
 shellescape==3.4.1
 simplejson==3.16.0
 six==1.11.0
-social-auth-core[openidconnect]==3.1.0+gx0
+social-auth-core[openidconnect]==3.1.0
```

Comment:
As above, this was fixed in commit fed80f19f6340f01401390c1ef5850e1fd71153a, need to re-run `make update-dependencies`.

Reply:
@nsoranzo Thanks, should be fixed now.